src/components/Home.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getData } from '../redux/home/home';
+import Card from './Card';
+
+export default function Home() {
```

Comment:
- [ ] The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), and therefore difficult to differentiate the details of each list item. Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that cells of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.
- [ ] Also note that the _details page_ has info arranged in alternating color pattern. Kindly ensure to improve upon the same


**Your design:**
![image](https://user-images.githubusercontent.com/12745474/159705943-57af9ef3-3a60-4a68-92f2-8351669b1f5e.png)

![image](https://user-images.githubusercontent.com/12745474/159706497-0e6e1341-e0db-4d38-a51b-27764b2491ee.png)



**The original design:**
![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)

![image](https://user-images.githubusercontent.com/12745474/159706704-f40d37a7-99f9-4758-9b06-d8a4b6b97341.png)

Reply:
@rloterh Thank you for your comment but I implemented the design as the requirement for the capstone project. The requirement doesn't include the design of dual color. I have implemented the barebone of the project and its functionality.